core/src/main/java/pl/netigen/core/netigenapi/NetigenFragment.kt

```diff
@@ -1,4 +1,17 @@
 package pl.netigen.core.netigenapi
 
-class NetigenFragment {
+import android.app.Activity
+import android.content.Context
+import androidx.fragment.app.Fragment
+
+abstract class NetigenFragment : Fragment() {
+
+    var netigenMainActivity: NetigenMainActivity<NetigenViewModel>? = null
```

Comment:
Why not nonnull lateinit?

Reply:
I think this class is not needed actually, for it introduces no real profit. Deleting it